contents/team/cameron-deleone.mdx

```diff
@@ -0,0 +1,11 @@
+---
+name: Cameron DeLeone
+jobTitle: Customer Success
+headshot: ../images/team/Cameron.png
+country: US
+startDate: 2021-11-29
+---
+
+I've jumped around a lot, professionally. Once upon a time, I studied meditation and the brain by training drug users to meditate and then putting them in an MRI machine. It was a lot of fun, but ultimately, I realized the academic thing was not for me.  After that, I cut my teeth at a few software companies, where I worked with a ton of users, helping them build all manner of fun and interesting things to automate their businesses.
+I found that I enjoyed making operations "just work" for businesses and developing relationships with customers quite a lot, so I spent the next couple of years consulting, doing just that. Eventually, I teamed up with a friend's agricultural business, where I built out an e-commerce store to sell poultry early in the pandemic, and built tools to improve operations. Ultimately, there are only so many technical problems to solve for a small poultry business, which is why I ended up at PostHog.
```

Comment:
@lottiecoxon Adding one more line break between each paragraph will make them into paragraphs on the site. (Right now it's just a single giant paragraph.)

![image](https://user-images.githubusercontent.com/154479/148649719-137164bd-df4d-40a0-be52-97c443aac128.png)